ballerina/Module.md

```diff
@@ -0,0 +1,101 @@
+## Overview
+
+[Guidewire InsuranceNow](https://www.guidewire.com/products/insurancenow) is a cloud-based insurance platform offering comprehensive tools for policy, billing, and claims management, designed to streamline operations and improve customer service in the insurance industry.
+
+The `ballerinax/guidewire.insnow` package provides a client API to connect to Guidewire InsuranceNow cloud API using Ballerina.
+
+## Compatibility
+
+|                        |   Version    |
+|:-----------------------|:------------:|
+| Guidewire InsuranceNow | **2023.3 (Innsbruck)** |
```

Comment:
IMO we dont need to have a single entry table of this. Instead we can mention the supported API version in the overview, similar to other connectors.

Reply:
Fixed with [3ffa97ac36318fcdd70cca1fec891f1694cb734c](https://github.com/ballerina-platform/module-ballerinax-guidewire.insnow/pull/12/commits/3ffa97ac36318fcdd70cca1fec891f1694cb734c)